src/server/models/users.model.js

```diff
@@ -8,6 +8,7 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
+  userId: String,
```

Comment:
<img width="1150" alt="2016-08-31 11 11 33" src="https://cloud.githubusercontent.com/assets/11007191/18113653/76cfd6e2-6f6c-11e6-976c-f3f65c18c1ab.png">

_id is created when user instance created so we can use _id value as id. But if we add userId field and copy the _id value after user instance created, _id and userId field would be duplicated!